Dockerfile

```diff
@@ -2,31 +2,23 @@
 # the Google Cloud Platform.
 
 # Install pipelines tool.
-FROM golang:1.10
-ENV GOPATH /go
-RUN go get github.com/googlegenomics/pipelines-tools/pipelines
-
+FROM golang:1.11
+RUN go get -ldflags '-extldflags "-fno-PIC -static"' -buildmode pie -tags 'osusergo netgo static_build' github.com/googlegenomics/pipelines-tools/pipelines
```

Comment:
You shouldn't really need to build this statically I wouldn't think...

Reply:
I copied this from VariantTransforms [Dockerfile](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/docker/Dockerfile) since that line failed the way it was (`go get github.com/...`).

I tried it with golang 1.11 after your comment and it worked. So I removed all the extra flags.